apps/modernization-ui/src/apps/search/investigation/GeneralSearchFields.tsx

```diff
@@ -0,0 +1,237 @@
+                name="jurisdiction"
+                onChange={() => 'void'}
+                options={[
+                    {
+                        label: 'Jurisdiction',
+                        name: 'coming soon',
+                        value: 'comingsoon'
+                    }
+                ]}
+            />
+            <Controller
+                control={form.control}
+                name="pregnancyStatus"
+                render={({ field: { name, value } }) => (
+                    <SelectInput
+                        data-testid="pregnancyStatus"
+                        name={name}
+                        id="pregnancyStatus"
+                        label="Pregnancy test"
+                        onChange={() => form.setValue('pregnancyStatus', value)}
+                        options={[
+                            { name: PregnancyStatus.Yes, value: PregnancyStatus.Yes },
+                            { name: PregnancyStatus.No, value: PregnancyStatus.No },
+                            { name: PregnancyStatus.Unknown, value: PregnancyStatus.Unknown }
+                        ]}
+                    />
+                )}
+            />
+            <Controller
+                control={form.control}
+                name="identification.type"
+                render={({ field: { value, name } }) => (
+                    <SelectInput
+                        name={name}
+                        defaultValue={asValue(value)}
+                        label="Event id type"
+                        dataTestid={name}
+                        htmlFor={name}
+                        options={Object.values(InvestigationEventIdType).map((event) => {
+                            return {
+                                name: formatInterfaceString(event),
+                                value: event
+                            };
+                        })}
+                    />
+                )}
+            />
+            <Controller
+                control={form.control}
+                name={'eventDate.type'}
+                render={({ field: { value, name } }) => (
+                    <SelectInput
+                        data-testid={name}
+                        name={name}
+                        value={asValue(value)}
+                        id="eventDateType"
+                        label="Event date type"
+                        options={Object.values(InvestigationEventIdType).map((event) => {
+                            return {
+                                label: formatInterfaceString(event),
+                                name: formatInterfaceString(event),
+                                value: event
+                            };
+                        })}
+                    />
+                )}
+            />
+
+            {watch.eventDate?.type ? (
+                <>
+                    <Controller
+                        control={form.control}
```

Comment:
suggestion: May need to add `shouldUnregister` to the `Controller` so that the values for `eventDate.from` and `eventDate.to` are cleared if the event type is directly cleared by the user.